docs/Coiote_DM_Device_Onboarding/Quick_start.md

```diff
@@ -0,0 +1,24 @@
+# Quick start
+
+Learn how to onboard a LwM2M device in just a few clicks.   
+
+## Prerequisites
+
+- An active Coiote DM user account with appropriate role assigned.
+- A LwM2M device (except for the simulated test device scenario).
+
+## Onboard a device via the Management server with PSK security mode
+
+1. Upon logging in to Coiote DM, you will see the **Add your LwM2M device** panel. If the panel does not show, from the navigation menu, select **Device Inventory** and click the **Add device** button.
```

Comment:
this screen will show only if the user hasn't added any device yet. Consider mentioning it